cmd/dockerd/daemon.go

```diff
@@ -515,6 +518,29 @@ func initRouter(opts routerOptions) {
 			grpcBackends = append(grpcBackends, b)
 		}
 	}
+
+	plugins := daemonplugin.Graph()
+	ctx := context.Background()
```

Comment:
should we change the signature to pass the context from `daemonCli.start()`?

Reply:
done